docs/index.rst

```diff
@@ -0,0 +1,42 @@
+Welcome to OCS-CI's documentation!
+==================================
+
+This is a documentation for the OCS-CI_ project.
+OCS-CI is a testing framework based on pytest to test Red Hat OCS features
+and functionality using AWS and other supported platforms.
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Package documentations:
+
+   ocs_ci modules <apidoc/modules.rst>
+
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+   code_review.md
```

Comment:
Will there be generated links to access these files from this document?

Reply:
Sphinx will generate clickable table of content as specified.